node/src/test/scala/com/wavesplatform/http/RestAPISettingsHelper.scala

```diff
@@ -22,7 +22,7 @@ trait RestAPISettingsHelper {
         s"""waves.rest-api {
            |  api-key-hash = $keyHash
            |  transactions-by-address-limit = $MaxTransactionsPerRequest
-           |  distribution-by-address-limit = $MaxAddressesPerRequest
+           |  distribution-address-limit = $MaxAddressesPerRequest
```

Comment:
is this necessary to rename existing config parameter? What about backward compatibility?

Reply:
this is typo in test config